apps/financial-aid/backend/src/app/modules/application/models/spouse.response.ts

```diff
@@ -8,5 +8,8 @@ export class SpouseResponse {
   hasFiles: boolean
 
   @ApiProperty()
-  spouseName?: string
+  applicantName?: string
+
+  @ApiProperty()
+  applicantSpouseEmail?: string
```

Comment:
Shouldn't this be `applicantName` and `applicantSpouseEmail`, that is no `?` since we return an empty string but not undefined from the service?